src/components/Card/Card.tsx

```diff
@@ -0,0 +1,17 @@
+import useNavigateToPage from '../../hooks/useNavigate';
+import { ICardData } from '../../interfaces/search-result.interface';
+import styles from './Card.module.css';
+
+export default function Card({ title, images, mal_id }: ICardData) {
```

Comment:
I remember a long discussion at Yandex about how to better declare components. 

This is what we agreed on
```tsx
type CardProps = {
   cards: ICard[]
}

export const Card: React.FC<CardProps> = ({ title, images, mal_id }) => {
   return <></>;
}
```

1. **Not using default exports** will guarantee the consistency of the component's name across the application
2. **Storing props close to components** adds the clarity of what component accepts without going to a sep file as well as not polluting global space with strange interfaces. Leave it for data models.
3. **Using React.FC<Props> construction** will make your live with typescript a bit easier. 

I suggest you adopt this approach as a framework for every single component you declare.

Reply:
thank you, already fixed it